psalm-baseline.xml

```diff
@@ -1039,11 +1037,11 @@
       <code>$parser-&gt;ArithmeticPrimary()</code>
       <code>$parser-&gt;ArithmeticPrimary()</code>
     </PossiblyInvalidPropertyAssignmentValue>
-    <PossiblyNullPropertyAssignmentValue occurrences="3">
-      <code>null</code>
-      <code>null</code>
-      <code>null</code>
-    </PossiblyNullPropertyAssignmentValue>
+    <PropertyNotSetInConstructor occurrences="3">
```

Comment:
Should we just ignore `PropertyNotSetInConstructor` for the whole `Functions` directory? After all, we deliberately work with the uninitialized state.

Reply:
`Functions` and maybe more, I will check.

Reply:
https://github.com/doctrine/orm/pull/10218